akka-docs/src/main/paradox/typed/persistence-active-active.md

```diff
@@ -0,0 +1,191 @@
+# Active-Active Event Sourcing
+
+@@@ warning
+
+This module is currently marked as @ref:[may change](../common/may-change.md) because it is a new feature that
+needs feedback from real usage before finalizing the API. This means that API or semantics can change without
+warning or deprecation period. It is also not recommended to use this module in production just yet.
+
+@@@
+
+Event sourcing with `EventSourcedBehavior`s is based on the single writer principle, which means that there can only be one active instance of a `EventSourcedBehavior` with a given `persistenceId`. Otherwise, multiple instances would store interleaving events based on different states, and when these events would later be replayed it would not be possible to reconstruct the correct state.
+
+This restriction means that in the event of network partitions, and for a short time during rolling re-deploys, `EventSourcedBehaviors`s are unavailable.
+
+
+Active-active event sourcing enables running multiple replicas of each entity. 
+There is automatic replication of every event persisted to all replicas.
+
+For instance, a replica can be run per:
+
+* Data Center 
+* Availability zone or rack
+
+The motivations are:
+
+* Redundancy to tolerate failures in one location and still be operational
+* Serve requests from a location near the user to provide better responsiveness
+* Balance the load over many servers
+
+However, the event handler must be able to **handle concurrent events** as when active-active is enabled
+there is no longer the single writer principal as there is with a normal `EventSourcedBehavior`.
+
+## Relaxing the single writer principal for availability
```

Comment:
```suggestion
## Relaxing the single writer principle for availability
```

Here I'm sure it's about relaxing a principle instead of relaxing the principal. Maybe with a massage and a cocktail. 🍸

Reply:
hah